lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -529,6 +529,23 @@ previewers.git_branch_log = defaulter(function(opts)
 end, {})
 
 previewers.git_stash_diff = defaulter(function(opts)
+  return previewers.new_buffer_previewer {
+    get_buffer_by_name = function(_, entry)
+      return entry.value
+    end,
+
+    define_preview = function(self, entry, _)
+      putils.job_maker({ 'git', '--no-pager', 'stash', 'show', '-p',  entry.value }, self.state.bufnr, {
+        value = entry.value,
+        bufname = self.state.bufname,
+        cwd = opts.cwd
+      })
+      putils.regex_highlighter(self.state.bufnr, 'diff')
+    end
+  }
+end, {})
+
+previewers.git_commit_diff = defaulter(function(opts)
```

Comment:
This whole block needs to go. Probably a automerge issue. It reimplements `git_commit_diff` which is no longer a thing. (And it actually uses a stash call :rofl:)